src/Handler/Table/Create/CreateTableHandler.php

```diff
@@ -41,7 +43,6 @@ public function __invoke(
         assert($command->getPath()->count() === 1, 'CreateTableCommand.path is required and size must equal 1');
         assert($command->getTableName() !== '', 'CreateTableCommand.tableName is required');
         assert($command->getColumns()->count() > 0, 'CreateTableCommand.columns is required');
-        assert($command->getPrimaryKeysNames()->count() === 0, 'CreateTableCommand.primaryKeysNames is not supported in Bigquery');
```

Comment:
odoborem len assert dalej sa nastavuje [] pretoze to BQ nepodporuje ale nemalo byt o spadnut aby to bolo co najvac podobne ostatnym implementaciam